File: app/src/main/java/com/rtukpe/fixtures/ui/home/DashboardMvpContract.java
package com.rtukpe.fixtures.ui.home;

import com.rtukpe.fixtures.di.annotations.PerActivity;
import com.rtukpe.fixtures.ui.base.MvpPresenter;

/**
 * Created by rtukpe on 21/03/2018.
 */

@PerActivity
public interface DashboardMvpContract<V extends DashboardMvpView> extends MvpPresenter<V> {

}

File: app/src/main/java/com/rtukpe/fixtures/ui/home/DashboardMvpContract.kt
package com.rtukpe.fixtures.ui.home

import com.rtukpe.fixtures.di.annotations.PerActivity
import com.rtukpe.fixtures.ui.base.MvpPresenter

/**
 * Created by rtukpe on 21/03/2018.
 */

@PerActivity
interface DashboardMvpContract<V : DashboardMvpView> : MvpPresenter<V>
